### DISCUSSION TITLE
主页面鼠标放到书签那里和点击书签显示的不一样。

### DISCUSSION THREAD

**Discussion by huu3301** [electerm/electerm#3042] [Q&A]
在主页面那里，鼠标放到⊕这里或者放在书签那里，显示ssh-config这个文件夹，这个不知道是系统的还是自己以前版本已经删除掉。
![1](https://github.com/electerm/electerm/assets/44219645/238ef35a-075e-41d7-9f69-83246f3d3c50)

但是点击书签里面却没有，请问这个该怎么去除掉这个文件夹或隐藏这个文件夹？或者说如何清除掉所有旧的配置？
![2](https://github.com/electerm/electerm/assets/44219645/022f9bec-9e32-49d2-9372-bc2a1c793263)

**Comment by zxdong262** [comment]
这是从.ssh目录读取的
